packages/react/src/components/HotspotEditorModal/HotspotEditorTooltipTab/__snapshots__/HotspotEditorTooltipTab.story.storyshot

```diff
@@ -30,7 +30,7 @@ exports[`Storybook Snapshot tests and console checks Storyshots 2 - Watson IoT E
               data-invalid={null}
             >
               <input
-                aria-describedby="tooltip-form-title-helper-text"
+                aria-describedby=""
```

Comment:
I see we are no longer passing along a value for `aria-descibedby` in all these snapshots. Is this something we should be providing here or surfacing to the user?